packages/starlight/components/SidebarSublist.astro

```diff
@@ -57,6 +57,11 @@ interface Props {
 		margin-inline-start: var(--sl-sidebar-item-padding-inline);
 		border-inline-start: 1px solid var(--sl-color-hairline-light);
 		padding-inline-start: var(--sl-sidebar-item-padding-inline);
+		overflow-wrap: break-word;
+	}
+
+	ul :global(.sl-badge) {
+		max-width: 100%;
 	}
```

Comment:
I think an alternative solution could be to not do that and use `overflow-wrap: anywhere;` instead of `overflow-wrap: break-word;` in the `<Badge/>` component. This would mean that we would not have to do that every time we use a badge in a similar context.

Any thoughts?

Reply:
Neat, it works pretty well! So obviously, this is way better in terms of usage since there's nothing to do while using the component.

I was a bit worried about [`overflow-wrap: anywhere` browser support](https://caniuse.com/?search=overflow-wrap%3A%20anywhere) for Safari which is only from 15.4.



But the team confirmed that other features used in Astro and/or Starlight are only working from Safari 15.4.
I ran some tests with BrowserStack with Safari 15.3 just to see how's the fallback in our specific use case, but it's impossible to say what's happening:

![Screenshot 2023-09-26 at 18 43 49](https://github.com/withastro/starlight/assets/17381666/3c80df6f-9e20-4868-aeab-7597d90fa51d)

I don't have access to any Safari 15.4, but from 16, everything's rendered as what you can see in your browser right now deployed by Netlify. Double-checked with various versions of Chrome and Firefox and seems to be good to go with this solution.